dbconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "host": "localhost",
-  "user": "import_user",
-  "password": "password",
+  "user": "root",
```

Comment:
You are right, I added this file to gitignore file; because every time I want to test my code; I have to change the dbconfig.json